service/templates/admin.html

```diff
@@ -97,6 +97,7 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 						<div>Other</div>
 						<li class="link" ng-click="scrollTo('misc')">Misc Settings</li>
 						<li class="link" ng-click="scrollTo('buckets')">Bucket Config</li>
+						<li class="link" ng-click="scrollTo('cost')">Cost Config</li>
```

Comment:
Did you tell this to Aldo? I'm not sure if they're still working on the admin page but I remember recently that we had to tell them of all new admin fields